### ISSUE TITLE
Filenames with spaces aren't recrypted 

### ISSUE THREAD

**Issue by pau1ie** [fanf2/regpg#8]
It looks like there is an issue with quoting.

```bash
$ regpg recrypt -r
grep: ./secrets/general/Visual: No such file or directory
grep: COBOL: No such file or directory
grep: Linux-UNIX: No such file or directory
grep: for: No such file or directory
grep: PeopleSoft.xml.asc: No such file or directory
grep: ./secrets/general/COBOL: No such file or directory
grep: Server: No such file or directory
grep: Runtime: No such file or directory
grep: for: No such file or directory
grep: PeopleSoft.xml.asc: No such file or directory
```

I was expecting these files to be recrypted, and but they are not:

```bash
$ ls -l secrets/general/[CV]*
-rw-rw-r--. 1 psh35 psh35 13695 Jan  8 12:17 'secrets/general/COBOL Server Runtime for PeopleSoft.xml.asc'
-rw-rw-r--. 1 psh35 psh35 11229 Jan  5 17:15 'secrets/general/Visual COBOL Linux-UNIX for PeopleSoft.xml.asc'
```

The files have not been recrypted. It looks like an issue with quoting.

**Comment by pau1ie** [comment]
Unfortunately I don't understand perl enough to work out what went wrong, but I was able to work around the issue by recrypting the files one at a time:
```
$ regpg recrypt secrets/general/COBOL\ Server\ Runtime\ for\ PeopleSoft.xml.asc 
$ regpg recrypt secrets/general/Visual\ COBOL\ Linux-UNIX\ for\ PeopleSoft.xml.asc 
```

**Comment by fanf2** [comment]
Thanks for the bug report!

My mistake was to use the classic shell pipeline `find | xargs grep` with the classic Unix "who would want spaces in filenames?!" flaw. I've replaced it with File::Find so it should work now.

This bug is fixed in the new release, regpg-1.12, available from https://dotat.at/prog/regpg/
